readinglist/errors.py

```diff
@@ -0,0 +1,86 @@
+from .utils import json
```

Comment:
we should be consistent between relaltive and absolute imports. Here you're doing both.